x-pack/plugins/observability/public/pages/overview/old_overview_page.tsx

```diff
@@ -286,12 +301,39 @@ function PageHeader({ handleGuidedSetupClick, onTimeRangeRefresh }: PageHeaderPr
         />
       </EuiFlexItem>
       <EuiFlexItem grow={false} style={{ alignItems: 'flex-end' }}>
-        <EuiButton color="text" iconType="wrench" onClick={handleGuidedSetupClick}>
-          <FormattedMessage
-            id="xpack.observability.overview.guidedSetupButton"
-            defaultMessage="Guided setup"
-          />
-        </EuiButton>
+        <EuiTourStep
```

Comment:
We don't have any other way to render a tour step? The tour step would be only visible when a user dismisses the progress guided setup that is going to be "only once" and we are rendering the `EuiTourStep` always when is probably not needed.

Reply:
I'm not sure of the overhead of adding it all the time, but I don't mind changing it.